features/admin.organizations.v1/components/meta-attribute-auto-complete.tsx

```diff
@@ -233,7 +233,9 @@ const MetaAttributeAutoComplete: FunctionComponent<MetaAttributeAutoCompleteProp
      * @see {@link https://github.com/mui/material-ui/issues/40250}
      */
     return (
-        <div className={ classNames("meta-attribute-autocomplete", { "error": hasErrors }) }>
+        <div
+            className={ classNames("meta-attribute-autocomplete", { "error": hasErrors }) }
+            data-componentid={ `${ _componentId }-autocomplete` }>
```

Comment:
Let's remove the underscore here, as we are now using the `componentId` prop. If there is a prop that isn't used in the component but still needs to be defined, we'll prefix it with an underscore to avoid ESLint errors.

Reply:
This is now updated.